api/proxy/proxy.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+
+package proxy;
+
+service Proxy {
+    rpc Backup (BackupRequest) returns (BackupReply) {}
+    rpc Restore (RestoreRequest) returns (RestoreReply) {}
+}
+
+message BackupRequest {
+    string cluster_identifier = 1;
+    google.protobuf.Timestamp backup_timestamp = 2;
+    bytes backup = 3;
+}
```

Comment:
The request here needs to embed an audience JWT, generated by the Kubernetes apiserver, which is then used by the proxy to validate the identity of the caller.

Is the plan to embed that without this message type?

Reply:
Would we not sent the JWT as a bearer token?

Reply:
On the actual request? If this is easy enough to do with gRPC, then sure. But from what I recall it isn't _that_ simple.. could definitely be wrong 😄 I'm not too bothered either way

Reply:
Seems relatively straightforward https://jbrandhorst.com/post/grpc-auth/